docs/redirects.yaml

```diff
@@ -0,0 +1,6 @@
+# Authoritative list of redirects we have configured in RTD.  See the
+# docs.nextstrain.org repo's README.md¹ for more information on maintaining
+# redirects.
+#
+# ¹ https://github.com/nextstrain/docs.nextstrain.org#configuring-redirects
+---
```

Comment:
We need to include the pre-existing redirects in RTD in this file, or they'll be deleted when `sync` is run!

You can see the pre-existing ones with

```
rtd projects nextstrain-augur redirects
```

and get them as JSON with:

```
rtd --json projects nextstrain-augur redirects
```

which can be lightly massaged into the `redirects.yaml` format.

Then double-check that the initial `redirects.yaml` matches state by doing a dry-run sync against it and checking that nothing's changed:

```
rtd projects nextstrain-augur redirects sync --dry-run -f redirects.yaml
```

Reply:
Done in [force-push](https://github.com/nextstrain/augur/compare/c8607cb633ff516b17b5c3abd9a9170bfccc2806..50425d962939f5511878b067201ddd83c61d889b).

<details>
<summary>
Output of `rtd projects nextstrain-augur redirects sync --dry-run -f docs/redirects.yaml` at bb9218940ff2959c998f531830ebdddea7d3e1e5
</summary>

```
          Created           
                            
  Type   From URL   To URL  
 ────────────────────────── 
                            
          Deleted           
                            
  Type   From URL   To URL  
 ────────────────────────── 
                            
Created 0, deleted 0, kept 13.

No changes made in --dry-run mode.  Pass --wet-run for realsies.
```

</details>

<details>
<summary>
Output of `rtd projects nextstrain-augur redirects sync --dry-run -f docs/redirects.yaml` at 50425d962939f5511878b067201ddd83c61d889b
</summary>

```
Creating: page /api/augur.export_v1.html → /api/developer/augur.export_v1.html
Creating: page /api/augur.utils.html → /api/developer/augur.utils.html
Creating: page /api/augur.align.html → /api/developer/augur.align.html
Creating: page /api/augur.util_support.node_data_reader.html → /api/developer/augur.util_support.node_data_reader.html
Creating: page /api/augur.lbi.html → /api/developer/augur.lbi.html
Creating: page /api/augur.distance.html → /api/developer/augur.distance.html
Creating: page /api/augur.export_v2.html → /api/developer/augur.export_v2.html
Creating: page /api/augur.frequency_estimators.html → /api/developer/augur.frequency_estimators.html
Creating: page /api/augur.index.html → /api/developer/augur.index.html
Creating: page /api/augur.ancestral.html → /api/developer/augur.ancestral.html
Creating: page /api/augur.util_support.node_data.html → /api/developer/augur.util_support.node_data.html
Creating: page /api/augur.titer_model.html → /api/developer/augur.titer_model.html
Creating: page /api/augur.util_support.html → /api/developer/augur.util_support.html
Creating: page /api/augur.util_support.node_data_file.html → /api/developer/augur.util_support.node_data_file.html
Creating: page /api/augur.frequencies.html → /api/developer/augur.frequencies.html
Creating: page /api/augur.tree.html → /api/developer/augur.tree.html
Creating: page /api/augur.clades.html → /api/developer/augur.clades.html
Creating: page /api/augur.sequence_traits.html → /api/developer/augur.sequence_traits.html
Creating: page /api/augur.titers.html → /api/developer/augur.titers.html
Creating: page /api/augur.parse.html → /api/developer/augur.parse.html
Creating: page /api/augur.traits.html → /api/developer/augur.traits.html
Creating: page /api/augur.mask.html → /api/developer/augur.mask.html
Creating: page /api/augur.util_support.color_parser.html → /api/developer/augur.util_support.color_parser.html
Creating: page /api/augur.import.html → /api/developer/augur.import.html
Creating: page /api/augur.io.html → /api/developer/augur.io.html
Creating: page /api/augur.util_support.color_parser_line.html → /api/developer/augur.util_support.color_parser_line.html
Creating: page /api/augur.refine.html → /api/developer/augur.refine.html
Creating: page /api/augur.filter.html → /api/developer/augur.filter.html
Creating: page /api/augur.html → /api/developer/augur.html
Creating: page /api/augur.validate.html → /api/developer/augur.validate.html
Creating: page /api/augur.export.html → /api/developer/augur.export.html
Creating: page /api/augur.reconstruct_sequences.html → /api/developer/augur.reconstruct_sequences.html
Creating: page /api/augur.util_support.date_disambiguator.html → /api/developer/augur.util_support.date_disambiguator.html
Creating: page /api/augur.measurements.html → /api/developer/augur.measurements.html
Creating: page /api/augur.version.html → /api/developer/augur.version.html
Creating: page /api/augur.translate.html → /api/developer/augur.translate.html
Creating: page /api/augur.validate_export.html → /api/developer/augur.validate_export.html
Creating: page /api/augur.filenames.html → /api/developer/augur.filenames.html
Creating: page /api/api.html → /api/developer/
                                                       Created                                                        
                                                                                                                      
  Type   From URL                                          To URL                                                     
 ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────── 
  page   /api/augur.export_v1.html                         /api/developer/augur.export_v1.html                        
  page   /api/augur.utils.html                             /api/developer/augur.utils.html                            
  page   /api/augur.align.html                             /api/developer/augur.align.html                            
  page   /api/augur.util_support.node_data_reader.html     /api/developer/augur.util_support.node_data_reader.html    
  page   /api/augur.lbi.html                               /api/developer/augur.lbi.html                              
  page   /api/augur.distance.html                          /api/developer/augur.distance.html                         
  page   /api/augur.export_v2.html                         /api/developer/augur.export_v2.html                        
  page   /api/augur.frequency_estimators.html              /api/developer/augur.frequency_estimators.html             
  page   /api/augur.index.html                             /api/developer/augur.index.html                            
  page   /api/augur.ancestral.html                         /api/developer/augur.ancestral.html                        
  page   /api/augur.util_support.node_data.html            /api/developer/augur.util_support.node_data.html           
  page   /api/augur.titer_model.html                       /api/developer/augur.titer_model.html                      
  page   /api/augur.util_support.html                      /api/developer/augur.util_support.html                     
  page   /api/augur.util_support.node_data_file.html       /api/developer/augur.util_support.node_data_file.html      
  page   /api/augur.frequencies.html                       /api/developer/augur.frequencies.html                      
  page   /api/augur.tree.html                              /api/developer/augur.tree.html                             
  page   /api/augur.clades.html                            /api/developer/augur.clades.html                           
  page   /api/augur.sequence_traits.html                   /api/developer/augur.sequence_traits.html                  
  page   /api/augur.titers.html                            /api/developer/augur.titers.html                           
  page   /api/augur.parse.html                             /api/developer/augur.parse.html                            
  page   /api/augur.traits.html                            /api/developer/augur.traits.html                           
  page   /api/augur.mask.html                              /api/developer/augur.mask.html                             
  page   /api/augur.util_support.color_parser.html         /api/developer/augur.util_support.color_parser.html        
  page   /api/augur.import.html                            /api/developer/augur.import.html                           
  page   /api/augur.io.html                                /api/developer/augur.io.html                               
  page   /api/augur.util_support.color_parser_line.html    /api/developer/augur.util_support.color_parser_line.html   
  page   /api/augur.refine.html                            /api/developer/augur.refine.html                           
  page   /api/augur.filter.html                            /api/developer/augur.filter.html                           
  page   /api/augur.html                                   /api/developer/augur.html                                  
  page   /api/augur.validate.html                          /api/developer/augur.validate.html                         
  page   /api/augur.export.html                            /api/developer/augur.export.html                           
  page   /api/augur.reconstruct_sequences.html             /api/developer/augur.reconstruct_sequences.html            
  page   /api/augur.util_support.date_disambiguator.html   /api/developer/augur.util_support.date_disambiguator.html  
  page   /api/augur.measurements.html                      /api/developer/augur.measurements.html                     
  page   /api/augur.version.html                           /api/developer/augur.version.html                          
  page   /api/augur.translate.html                         /api/developer/augur.translate.html                        
  page   /api/augur.validate_export.html                   /api/developer/augur.validate_export.html                  
  page   /api/augur.filenames.html                         /api/developer/augur.filenames.html                        
  page   /api/api.html                                     /api/developer/                                            
                                                                                                                      
          Deleted           
                            
  Type   From URL   To URL  
 ────────────────────────── 
                            
Created 39, deleted 0, kept 13.

No changes made in --dry-run mode.  Pass --wet-run for realsies.
```

</details>